src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -1,20 +1,20 @@
 package frc.robot.subsystems.shooter;
 
 public interface Shooter {
-  /**
-   * Disable the speed of a motors for the shooter.
-   */
+  /** Disable the voltage of a motors for the shooter. */
   public default void disable() {}
```

Comment:
You probably don't want to declare this as "default" since you likely always want who ever implements the Shooter interface to always implement disable().  Same goes for enable() and setVoltage().